app/models/link.rb

```diff
@@ -21,10 +21,13 @@ def self.filter_editions(scope, filters)
       logger.warn("filter_editions called with multiple filters. These will be ANDed together in a way that probably isn't what we want. Filters were: #{filters.inspect}")
     end
 
-    scope = scope.joins(document: :link_set_links)
+    scope = scope.joins("LEFT JOIN links edition_links ON edition_links.edition_id = editions.id")
```

Comment:
Might be worth adding the context to the commit msg as to why this change is happening, i.e. because embedded content will be stored as an Edition link